rest_test/run_all_tests_from_make.sh

```diff
@@ -0,0 +1,109 @@
+#! /bin/bash
+# TODO import from development environment in envkey
+password="password"
+validatorMnemonic="equip town gesture square tomorrow volume nephew minute witness beef rich gadget actress egg sing secret pole winter alarm law today check violin uncover"
+
+#   address: kava1ffv7nhd3z6sych2qpqkk03ec6hzkmufy0r2s4c
+#   address: kavavaloper1ffv7nhd3z6sych2qpqkk03ec6hzkmufyz4scd0
+
+faucet="chief access utility giant burger winner jar false naive mobile often perfect advice village love enroll embark bacon under flock harbor render father since"
+
+# address: kava1ls82zzghsx0exkpr52m8vht5jqs3un0ceysshz
+# address: kavavaloper1ls82zzghsx0exkpr52m8vht5jqs3un0c5j2c04
+
+# Remove any existing data directory
+rm -rf ~/kavatmp
+
+# create validator key
+printf "$password\n$validatorMnemonic\n" | kvcli keys add vlad --recover --home ~/kavatmp
+# create faucet key
+printf "$password\n$faucet\n" | kvcli --home ~/kavatmp keys add faucet --recover --home ~/kavatmp
+
+# function used to show that it is still loading
+showLoading() {
+  mypid=$!
+  loadingText=$1
+
+  echo -ne "$loadingText\r"
+
+  while kill -0 $mypid 2>/dev/null; do
+    echo -ne "$loadingText.\r"
+    sleep 0.5
+    echo -ne "$loadingText..\r"
+    sleep 0.5
+    echo -ne "$loadingText...\r"
+    sleep 0.5
+    echo -ne "\r\033[K"
+    echo -ne "$loadingText\r"
+    sleep 0.5
+  done
+
+  echo "$loadingText...finished"
+}
+
+# build the go setup test file
+rm -f rest_test/test
+go build rest_test/debugging_tools/test.go & showLoading "Building go test file, please wait"
+
+# Create new data directory
+{
+kvd --home ~/kavatmp init --chain-id=testing vlad # doesn't need to be the same as the validator
+} > /dev/null 2>&1
+kvcli --home ~/kavatmp config chain-id testing # or set trust-node true
```

Comment:
`kvd` and `kvcli` should have their own separate home folders. Doesn't look like it's actually causing problems here, but it could trigger weird bugs in the future if they're overwritting each other's files.

Couple of suggestions
- if you define a variable like `kvdHome=/some/path` then pass flags like `--home $kvdHome` it will be easier to change the folder location in the future.
- If you put the home folders in `/tmp`, then they won't clutter or overwrite anything in the use's home directory, and they'll be cleaned up automatically by the os.

Reply:
🆗